### PR TITLE
"failed to update live settings" message

### PR DESCRIPTION
**Pull request by darkbarker**
if there are a lot of live settings, then no-noticeable error message causes serious inconvenience ( see issue #464  )

### CODE REVIEW

**Comment by camilonova** [comment]
If the form is not valid you will get a proper error message, no need to add an extra one.

**Comment by darkbarker** [comment]
> If the form is not valid you will get a proper error message, no need to add an extra one.

@camilonova sorry but you didnt understand what problem the patch solves,  please read issue #464 where the problem is described in detail.

in short: error message (it exists, thats right) with many settings is absolutely unusable.

only difference between success and error is:
* success: green banner
* error: silent absence ("error message" is far at the bottom of the page)
imho this is a bad design, we have real cases when its not noticed

**Comment by camilonova** [comment]
@darkbarker Fix the build problems and I'll be happy to merge this.

File: constance/locale/uk/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -61,6 +61,10 @@ msgstr ""
 msgid "Live settings updated successfully."
 msgstr "Налаштування успішно збережені."
 
+#: admin.py:267
+msgid "Failed to update live settings."
+msgstr "Не вдалося зберегти налаштування"
```
Comment (Mogost) [inline constance/locale/uk/LC_MESSAGES/django.po:66]:
```suggestion
msgstr "Не вдалося зберегти налаштування."
```

File: constance/locale/ru/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -49,6 +49,10 @@ msgstr ""
 msgid "Live settings updated successfully."
 msgstr "Настройки успешно сохранены"
 
+#: admin.py:267
+msgid "Failed to update live settings."
+msgstr "Не удалось сохранить настройки"
```
Comment (Mogost) [inline constance/locale/ru/LC_MESSAGES/django.po:54]:
```suggestion
msgstr "Не удалось сохранить настройки."
```

**Comment by darkbarker** [comment]
@camilonova i fixed build (tests, as i understand)

**Comment by Mogost** [comment]
@darkbarker Could you please rebuild `.mo` files after the translation change?

**Review by camilonova — APPROVED** [review]

**Comment by darkbarker** [comment]
@Mogost done, rebuilded (sorry for wait)

**Pull request merged**